docs/source/faq_sby.rst

```diff
@@ -138,6 +138,81 @@ This includes:
 - multiple edges of the same clock signal
 - any asynchronous logic (with the exception of asynchronous resets that should be treated as synchronous)
 
+Handling combinational loops
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Q:** SBY is failing to process my design, reporting that it has a topological logic loop even
+if I don't think there is one. What can I do to allow SBY to process the design?
+
+**A:** Formal verification tools struggle to handle combinational loops as the underlying solvers
+only allow nets to take a single value per clock cycle, a property that is violated by unstable
+combinational loops that oscillate. Even when you don't explicitly include a combinational loop
+in the design, they can be introduced by multi-clock mode which introduces combinational
+paths from clock and reset inputs to the Q output for flip-flops and from the enable input to the
+output of latches.
+
+If you are confident that a loop is stable, you can break the loop by replacing a connection with
+an assumption. This forces the solvers to only ever produce counterexamples where the loop is
```

Comment:
Can we be more explicit that this can place restrictions on the transitive input cone of the loop that in turn can affect properties that share some of the input cone even if the broken loop itself is not in the input cone of the property? I'm not sure if there's a good way to do that without distracting too much from the actual issue, so maybe this should be a separate FAQ entry on potential pitfalls of assumptions and this could then refer to?

Reply:
Yeah I think that makes sense as a separate entry that gets referenced